bciers/apps/compliance/src/app/components/compliance-summaries/ComplianceSummariesPage.tsx

```diff
@@ -14,28 +17,48 @@ export default async function ComplianceSummariesPage({
   const { reporting_year: reportingYear } = await getReportingYear();
   const initialData = await fetchComplianceSummariesPageData(searchParams);
   const frontEndRole = await getSessionRole();
+  const currentDate = new Date(); // Current date
+  const dueDate = new Date(reportingYear + 1, 10, 30, 23, 59, 59); // November 30 of the reporting year + 1
   const isAllowedCas = [
     FrontEndRoles.CAS_DIRECTOR,
     FrontEndRoles.CAS_ANALYST,
     FrontEndRoles.CAS_ADMIN,
     FrontEndRoles.CAS_VIEW_ONLY,
   ].includes(frontEndRole);
+  const isAllowedIndustryUser = [
+    FrontEndRoles.INDUSTRY_USER,
+    FrontEndRoles.INDUSTRY_USER_ADMIN,
+  ];
+  const obligationsNotMet = initialData.rows.some(
+    (row) =>
+      row.obligation_id &&
+      row.status === ComplianceSummaryStatus.OBLIGATION_NOT_MET,
+  );
+
   return (
     <>
       <div className="mb-5">
         <div className="mb-2">
-          <AlertNote>
-            Your compliance obligation for the {reportingYear} reporting year is{" "}
-            <strong>due on November 30, {reportingYear + 1}</strong>. Please pay
-            five business days in advance to account for the processing time.
-          </AlertNote>
+          {isAllowedIndustryUser && obligationsNotMet && (
+            <AlertNote>
+              Your compliance obligation for the {reportingYear} reporting year{" "}
+              {currentDate > dueDate ? "was" : "is"}{" "}
+              <strong>due on November 30, {reportingYear + 1}</strong>. Please
+              pay five business days in advance to account for the processing
+              time.
+            </AlertNote>
+          )}
         </div>
-        <AlertNote>
-          An automatic overdue penalty has been incurred and{" "}
-          <strong>accrues at 0.38% daily</strong> since the compliance
-          obligation was not paid by its due date. You may pay the penalty after
-          the compliance obligation is paid.
-        </AlertNote>
+        {isAllowedIndustryUser &&
+          currentDate > dueDate &&
+          obligationsNotMet && (
```

Comment:
I think we additionally need to check if there's a penalty. From Nan's comments in the card (`when the obligation deadline has passed AND there is penalty`), it sounds like this alert should show if the penalty is unpaid, even if the obligation is paid